tests/test_utils_reqser.py

```diff
@@ -25,7 +25,8 @@ def test_all_attributes(self):
             cookies={'currency': u'руб'},
             encoding='latin-1',
             priority=20,
-            meta={'a': 'b'})
+            meta={'a': 'b'},
+            flags= ['testFlag'])
```

Comment:
There is an extra space after `=` - could you plese remove it? We're trying to follow PEP8 in Scrapy source code, with 80 columns limit somewhat relaxed.